### PR TITLE
added tini to prevent zombies caused by nodemon

### PR DESCRIPTION
**Pull request by morph027**
The clean PR without the dnsmasq stuff ;)

tini cleanly handles the exit of childs, so no zombies left afterwards.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,15 +22,20 @@ COPY src/package.json ${GITLAB_CE_PAGES_WEBHOOK_DIR}/
...
 
+ENV TINI_VERSION v0.10.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
+
 COPY entrypoint.sh /
 COPY config/nginx.conf /etc/nginx/nginx.conf
 RUN rm -f /etc/nginx/conf.d/*
+RUN chown -R nginx: /var/log/nginx
```
Comment (YuMS) [inline Dockerfile]: What is this for?

Comment (YuMS) [inline Dockerfile]: What is this for?

Comment (morph027) [reply Dockerfile]: Cosmetics, nginx was complaining about not being able to log output.

Comment (YuMS) [reply Dockerfile]: According to https://blog.docker.com/2015/04/tips-for-deploying-nginx-official-image-with-docker/ NGINX image will redirect `access.log` to `/dev/stdout`, and redirect `error.log` to `/dev/stderr`. Why is it complaining?
image

Comment (morph027) [reply Dockerfile]: Hm, then i guess i was mislead by the file or directory error not found. In this case, if output still is being shown, everything's alright.

**Pull request merged**